code/game/gamemodes/gang/gang_things.dm

```diff
@@ -40,7 +41,8 @@
 		to_chat(user, policy)
 	team_to_use.add_member(user.mind)
 	for(var/threads in team_to_use.free_clothes)
-		new threads(get_turf(user))
+		var/obj/O = new threads(get_turf(user))
+		O.armor = list("melee" = 7.5, "bullet" = 7.5, "laser" = 7.5, "energy" = 7.5, "bomb" = 7.5, "bio" = 7.5, "rad" = 7.5, "fire" = 7.5, "acid" = 7.5, "wound" = 7.5)
```

Comment:
This is not how armor works.

Reply:
Yes it is? Armor stacking is getting removed soon.

Reply:
> Armor stacking is getting removed soon.

Excuse me?

Reply:
Armor gets converted from list to datum, is what nemvar is saying.
Use the armor procs, don't just assign a list to the variable outside of object definition.